unity-renderer/Assets/Scripts/MainScripts/DCL/Analytics/GenericAnalytics/GenericAnalytics.cs

```diff
@@ -4,7 +4,7 @@ public static class GenericAnalytics
 {
     public static void SendAnalytic(string eventName, Dictionary<string, string> data)
     {
-        IAnalytics analytics = DCL.Environment.i.platform.serviceProviders.analytics;
+        IAnalytics analytics = DCL.Environment.i?.platform?.serviceProviders?.analytics;
```

Comment:
out of curiosity, why the need for this change?

Reply:
Its a design flaw. My tests where failling because this static environment was giving a NRE. I had to add the null check.

I couldnt find an wasy way to refactor and inject this analytics provider, and its out of scope from this ticket. So I added the null check.